README.md

```diff
@@ -124,6 +124,7 @@ However, these same commands can be directed to operate locally as well via
 the `--local` or `-L` command (or setting `KO_DOCKER_REPO=ko.local`).  See
 the [`minikube` section](./README.md#with-minikube) for more detail.
 
+The image when you use `ko` to create won't set the create timestamp in the image attribute. If you want to set the create timestamp, just execute SOURCE_DATE_EPOCH=`date +%s` on your environment before use `ko`.
```

Comment:
```suggestion
In order to support [reproducible builds](https://reproducible-builds.org), `ko` doesn't embed timestamps in the images it produces by default; however, `ko` does respect the [`SOURCE_DATE_EPOCH`](https://reproducible-builds.org/docs/source-date-epoch/) environment variable.

For example, you can set this to the current timestamp by executing:

    export SOURCE_DATE_EPOCH=`date +%s

or to the latest git commit's timestamp with:

    export SOURCE_DATE_EPOCH=$(git log -1 --format='%ct')
```

Reply:
Thank you for your review. I have already updated the readme. @jonjohnsonjr